recipes/djangorestframework-gis/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set name = "djangorestframework-gis" %}
+{% set version = "0.10.1" %}
+{% set sha256 = "ae255dc287b08eb4b6cc5bcc2b0af23cf8ba7811f4cc0152d7553914d0ef22dd" %}
+{% set build = 0 %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://github.com/djangonauts/django-rest-framework-gis/archive/v{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: {{ build }}
+  script: python setup.py install --single-version-externally-managed --record=record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - djangorestframework >=3.3
+
+  run:
+    - python
+    - djangorestframework >=3.3
+
+test:
+  imports:
+    - rest_framework_gis
+
+about:
+  home: https://github.com/djangonauts/django-rest-framework-gis
+  summary: 'Geographic add-ons for Django Rest Framework'
+  license: MIT
+  license_family: MIT
+  license_file: LICENSE
+  doc_url: https://github.com/djangonauts/django-rest-framework-gis
+  dev_url: https://github.com/djangonauts/django-rest-framework-gis
+
+extra:
+  recipe-maintainers:
+    - kwilcox
```

Comment:
@kwilcox feel free to add me to those so I can keep track later and update the feedstock.